WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -2738,6 +2737,14 @@ static GSourceFuncs _handlerIntervention =
             }
 
             _adminLock.Unlock();
+
+            TRACE_L1("%s and _httpStatusCode : %d ", URL.c_str(), _httpStatusCode);
```

Comment:
Please, modify first print message in OnLoadFinished() function to include _httpStatusCode instead of having new print.

Reply:
yes, I will do it.